anxcloud/resource_lbaas_loadbalancer.go

```diff
@@ -0,0 +1,100 @@
+package anxcloud
+
+import (
+	"context"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"go.anx.io/go-anxcloud/pkg/api"
+	lbaasv1 "go.anx.io/go-anxcloud/pkg/apis/lbaas/v1"
+)
+
+func resourceLBaaSLoadBalancer() *schema.Resource {
+	return &schema.Resource{
+		Description:   "This resource allows you to create and manage LBaaS LoadBalancer resources.",
+		CreateContext: tagsMiddlewareCreate(resourceLBaaSLoadBalancerCreate),
+		ReadContext:   tagsMiddlewareRead(resourceLBaaSLoadBalancerRead),
+		UpdateContext: tagsMiddlewareUpdate(resourceLBaaSLoadBalancerUpdate),
+		DeleteContext: resourceLBaaSLoadBalancerDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(5 * time.Minute),
+			Read:   schema.DefaultTimeout(time.Minute),
+			Update: schema.DefaultTimeout(5 * time.Minute),
+			Delete: schema.DefaultTimeout(5 * time.Minute),
```

Comment:
we don't need that high timeout values, let's go with 30s for each except delete (in case it ever learns recursive deletion of resources)?